README.md

```diff
@@ -530,8 +530,48 @@ See [Confluence TOC Macro] for the list of parameters - keep in mind that here
 they start with capital letters. Every skipped field will have the default
 value, so feel free to include only the ones that you require.
 
-
 [Confluence TOC Macro]:https://confluence.atlassian.com/conf59/table-of-contents-macro-792499210.html
+
+### Insert PageTree
+
+```markdown
+# My First Heading
+<!-- Include: ac:pagetree -->
+```
+
+The pagetree macro works almost the same as the TOC above, but the tree behavior
+is more desireable for creating placeholder pages above collectiosn of SOPs.
```

Comment:
```suggestion
is more desirable for creating placeholder pages above collections of SOPs.
```

What is SOP in this context?

Reply:
I could have wrote documents.
I use it to organize standard operating procedure documentation.